website/sidebars.js

```diff
@@ -70,6 +70,9 @@ module.exports = {
       'developer/research/research-bamboo'
 
     ],
+    "API": [
```

Comment:
Update to `"API Reference"` and move block between the `Deployment Guide` and `Research`

Reply:
moved